.circleci/config.yml

```diff
@@ -17,34 +17,51 @@ jobs:
     docker:
       - image: cimg/openjdk:8.0
     <<: *lint_steps
+
+  # JDK 8 (LTS) - 31 Dec 2030 (extended support)
   test_openjdk8:
     docker:
       - image: cimg/openjdk:8.0
     <<: *build_steps
+
+  # JDK 11 (LTS) - 31 Jan 2032 (extended support)
   test_openjdk11:
     docker:
       - image: cimg/openjdk:11.0
     <<: *build_steps
-  test_openjdk15:
-    docker:
-      - image: cimg/openjdk:15.0
-    <<: *build_steps
+
+  # JDK 11 (LTS) - 30 Sep 2029 (extended support)
   test_openjdk17:
     docker:
       - image: cimg/openjdk:17.0
     <<: *build_steps
+
+  # Available in Ubuntu 22.04
```

Comment:
What does this comment mean? Is jdk 18 what ships by default with ubuntu 22.04?

Reply:
I was looking at which ones we test with + which ones are in the LTS version of Ubuntu. Mainly as trying to figure out what the test matrix should look like.